data/json/items/containers/generic.json

```diff
@@ -43,5 +43,26 @@
     "use_action": { "type": "deploy_furn", "furn_type": "f_trashcan" },
     "symbol": ")",
     "color": "light_gray"
+  },
+  {
+    "id": "trashcan_large",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "industrial trash can" },
+    "looks_like": "f_trashcan",
+    "description": "A really big trashcan in which you can store, you guessed it, trash, ready for the garbage truck to pick up.  Made out of thick plastic, this can was meant for household or industrial use, something you roll up to the curb once a week.",
+    "weight": "4535 g",
+    "//1": "Weight taken from https://www.uline.com/Product/Detail/H-7938BL/Trash-Cans-with-Wheels/Uline-Trash-Can-with-Wheels-95-Gallon-Black?pricode=WB3072&utm_source=Bing&utm_medium=pla&utm_term=H-7938BL&utm_campaign=Facilities%2BMaintenance&utm_source=Bing&utm_medium=pla&utm_term=H-7938BL&utm_campaign=Facilities%2BMaintenance&msclkid=d3c8827d02691eb87066e72c65ff5790",
+    "volume": "400 L",
+    "longest_side": "114 cm",
+    "price": "150 USD",
+    "//2": "Based on above link.",
+    "pocket_data": [
+      { "pocket_type": "CONTAINER", "max_contains_volume": "38 L", "open_container": true, "rigid": true, "watertight": true }
```

Comment:
about 10% of the total volume is used for storage, in this case I would just give the pocket about 90% of the total volume for storage ~360L or so, also the link mentions the capacity 330lbs about 149kg, please I would love to have that